<?xml version="1.0"?>
<launch>

  <arg name="CAM" default="3"/>
  <arg name="markertype" default="bch"/>

  <node type="icl_node" name="icl_node$(arg CAM)_2" pkg="icl" respawn="false" output="screen">
    <param name="topic_in_image" type="string" value="/genicam_cam$(arg CAM)/image_rect"/>
    <param name="image_camera_info" type="string" value="/genicam_cam$(arg CAM)/camera_info"/>
    <param name="topic_out_odom" type="string" value="/cam/odom"/>
    <param name="fps" value="21"/>
    <param name="mono_color" value="1"/>
    <param name="marker_type" value="$(arg markertype)"/>
    <param name="marker_size" value="100"/>
    <param name="fid_config" value="$(find icl)/config/fid_config.xml"/>
    <param name="focal_length" value="8"/>
    <param name="draw3d_axis" value="0"/>
    <param name="verbose" value="1"/>
  </node>
</launch>
